examples/src/main/java/com/traveltime/examples/Utils.java

```diff
@@ -0,0 +1,35 @@
+package com.traveltime.examples;
+
+import com.traveltime.sdk.dto.common.Coordinates;
+import javafx.util.Pair;
+
+import org.locationtech.jts.geom.Point;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Utils {
+    private static Random rd = new Random();
+
+    private static Coordinates generateCoordinates(Coordinates center, double range) {
+        return new Coordinates(generateDouble(center.getLat(), range), generateDouble(center.getLng(), range));
+    }
+
+    private static double generateDouble(double center, double range) {
+        return center + (rd.nextDouble() - 0.5) * range;
+    }
```

Comment:
This could be in generateCoordinates method

Reply:
it'll be a duplication